src/include/log.h

```diff
@@ -87,6 +87,12 @@ union __wt_lsn {
 #define	WT_LOGC_KEY_FORMAT	WT_UNCHECKED_STRING(III)
 #define	WT_LOGC_VALUE_FORMAT	WT_UNCHECKED_STRING(qIIIuu)
 
+/*
+ * Size range for the log files.
+ */
+#define	WT_LOG_FILE_MAX ((uint64_t)2 * WT_GIGABYTE)
+#define	WT_LOG_FILE_MIN (100 * 1024)
```

Comment:
Minor nit: change `1024` to `WT_KILOBYTE`.

Reply:
Done.